docs/node-operators/archive-nodes/getting-started.mdx

```diff
@@ -17,7 +17,7 @@ For some use cases, it is useful to maintain this historical data on an archive
 
 :::tip
 
-A zkApp can retrieve events and actions from one or more Mina archive nodes. If your smart contract needs to fetch events and actions from an archive node, see [How to Fetch Events and Actions](../zkapps/o1js/fetch-events-and-actions).
+A zkApp can retrieve events and actions from one or more Mina archive nodes. If your smart contract needs to fetch events and actions from an archive node, see [How to Fetch Events and Actions](../../zkapps/o1js/fetch-events-and-actions).
```

Comment:
you can prevent link errors by using https://github.com/o1-labs/docs2/wiki/Markdown-Syntax#links 

(omit the relative link .. for paths)